wire-ios/Wire-iOS Tests/SecurityLevel/SecurityLevelViewTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+import WireUtilities
+@testable import Wire
+
+final class SecurityLevelViewTests: ZMSnapshotTestCase {
+    private var callingUIFlag: DeveloperFlag!
+    private var deprecatedUIFlagStateBackup: Bool!
+
+    private var sut: SecurityLevelView!
+
+    override func setUp() {
+        super.setUp()
+        callingUIFlag = DeveloperFlag.deprecatedCallingUI
+        deprecatedUIFlagStateBackup = callingUIFlag.isOn
+        callingUIFlag.isOn = false
+        sut = SecurityLevelView()
+        sut.backgroundColor = .white
```

Comment:
Changing this one 👍 
In case of dark & light modes it's no need to change it directly on `overrideInterfaceStyle`, as it all logic for that happens in `verifyInAllColorSchemes`.